src/VisualStudio/Core/Def/Implementation/Remote/JsonRpcClient.cs

```diff
@@ -195,11 +193,5 @@ private void OnDisconnected(object sender, JsonRpcDisconnectedEventArgs e)
             // tell we got disconnected
             Disconnected(e);
         }
-
-        public void Dispose()
-        {
-            Dispose(disposing: true);
-            GC.SuppressFinalize(this);
```

Comment:
do we not need this code anymore in our Dispose impl.  why/why not?

Reply:
The new dispose pattern does not _allow_ user-defined finalizers, so `GC.SuppressFinalize` is always a NOP.

Reply:
I'll follow up on switching to the alternative dispose pattern across the IDE code base in a separate PR.